zk_toolbox/crates/zk_inception/src/commands/chain/set_token_multiplier_setter.rs

```diff
@@ -0,0 +1,115 @@
+use anyhow::Context;
+use common::{
+    config::global_config,
+    forge::{Forge, ForgeScript, ForgeScriptArgs},
+    logger,
+    spinner::Spinner,
+};
+use config::{forge_interface::script_params::ACCEPT_GOVERNANCE_SCRIPT_PARAMS, EcosystemConfig};
+use ethers::{abi::parse_abi, contract::BaseContract, utils::hex};
+use lazy_static::lazy_static;
+use xshell::Shell;
+use zksync_basic_types::{Address, H256};
+
+use crate::{
+    messages::{
+        MSG_CHAIN_NOT_INITIALIZED, MSG_L1_SECRETS_MUST_BE_PRESENTED,
+        MSG_TOKEN_MULTIPLIER_SETTER_UPDATED_TO, MSG_UPDATING_TOKEN_MULTIPLIER_SETTER_SPINNER,
+    },
+    utils::forge::{check_the_balance, fill_forge_private_key},
+};
+
+lazy_static! {
+    static ref SET_TOKEN_MULTIPLIER_SETTER: BaseContract = BaseContract::from(
+        parse_abi(&[
+            "function chainSetTokenMultiplierSetter(address chainAdmin, address target) public"
+        ])
+        .unwrap(),
+    );
+}
+
+pub async fn run(args: ForgeScriptArgs, shell: &Shell) -> anyhow::Result<()> {
+    let chain_name = global_config().chain_name.clone();
+    let ecosystem_config = EcosystemConfig::from_file(shell)?;
+    let chain_config = ecosystem_config
+        .load_chain(chain_name)
+        .context(MSG_CHAIN_NOT_INITIALIZED)?;
+    let contracts_config = chain_config.get_contracts_config()?;
+    let l1_url = chain_config
+        .get_secrets_config()?
+        .l1
+        .context(MSG_L1_SECRETS_MUST_BE_PRESENTED)?
+        .l1_rpc_url
+        .expose_str()
+        .to_string();
+    let token_multiplier_setter_address = ecosystem_config
+        .get_wallets()
+        .unwrap()
```

Comment:
this can be a meaningful error, the function returns `anyhow::Result<()>` anyway

Reply:
good point - fixed.